app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,20 @@ def show
     @post = @user.posts.find(params[:id])
     @comments = @post.comments
   end
+
+  def new
```

Comment:
- [ ] I am not able to render the form for creating a post for a user. Please make sure this form renders. I would also suggest a button that enables a user to add a post.
![image](https://github.com/gdumani/blogapp/assets/45460192/449ef4e2-213e-4f5c-a9e1-3a79196d111d)

Reply:
The form for posting a new comment is valid for the current user and the URL has the format 
localhost:3000/post/new
There is no need to add a user/id before post/new since current user is known in the session!
This URL is invalid by design
![image](https://github.com/gdumani/blogapp/assets/60476910/446c4d76-8ca5-4116-adbe-b5ed7b8cce20)

The link to add a new user is at the top of the home screen 
![blog-app-home](https://github.com/gdumani/blogapp/assets/60476910/c92a063c-b64e-4f8b-940f-ca9152a3374e)